Makefile

```diff
@@ -36,12 +36,13 @@ BUILDER_IMAGE = public.ecr.aws/bottlerocket/bottlerocket-sdk-$(BOTTLEROCKET_SDK_
 
 export CARGO_HOME = $(TOP)/.cargo
 
-image: build brupop-image
+image: check-licenses brupop-image
```

Comment:
Removes the duplicate build step here. No reason to build binaries locally when building the image does the same thing. Calls into the `check-license` path to ensure licenses are good. This is the default target.